src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -93,12 +122,48 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
     }
 
     async deinitialize(): Promise<InternalCheckoutSelectors> {
+        this._deinitializePollingTimer();
         this._orderId = undefined;
         this._paypalCommercePaymentProcessor.deinitialize();
 
         return Promise.resolve(this._store.getState());
     }
 
+    private _initializePollingMechanism(submitForm: () => void, gatewayId?: string) {
+        if (gatewayId === PaymentStrategyType.PAYPAL_COMMERCE_ALTERNATIVE_METHODS) {
+            this._pollingInterval = window.setTimeout(async () => {
+                try {
+                    if (this._pollingTimer !== undefined) {
+                        this._pollingTimer += POLLING_INTERVAL;
+                    }
+                    const res = await this._paypalCommerceRequestSender.getOrderStatus();
+                    if (res.status.toLowerCase() !== ORDER_STATUS_CREATED && res.status.toLowerCase() !== ORDER_STATUS_APPROVED) {
```

Comment:
Looks like we can avoid using the `toLowerCase` method if we change the `status` constants according to actual status values.

for example: `'approved' => `APPROVED`` (suppose that status is uppercased)

Reply:
fixed